models/isam/vlans/isam_vlans.yml

```diff
@@ -0,0 +1,605 @@
+          - default = inherit
+          - inherit               ! inherit new-secure-forwarding
+          - disable               ! disable new-secure-forwarding
+          - enable                ! enable new-secure-forwarding
+          type: str
+          choices:
+          - inherit
+          - enable
+          - disable
+        aging-time:
+          description: 
+          - configure MAC aging time in seconds; in case of default,the system-level value is applicable.
+          - default = -1
+          type = int
+        l2cp-transparent:
+          description: enable l2cp-transparent
+          type: bool
+        in-qos-prof-name:
+          description:
+          - QoS ingress profile name
+          - default = default
+          type: str
+        ipv4-mcast-ctrl:
+          description:
+          - enable ipv4 multicast control: forward ipv4 multicast frames in this VLAN
+          type: bool
+        ipv6-mcast-ctrl:
+          description:
+          - enable ipv6 multicast control: forward ipv6 multicast frames in this VLAN
+          type: bool
+        mac-mcast-ctrl:
+          description:
+          - enable mac multicast control: forward mac multicast frames in this VLAN
+          type: bool
+        dis-proto-rip:
+          description:
+          - disable rip-ipv4 protocol
+          type: bool
+        proto-ntp:
+          description:
+          - enable ntp protocol
+          type: bool
+        dis-ip-antispoof:
+          description:
+          - disable IP anti-spoofing
+          type: bool
+        unknown-unicast:
+          description:
+          - enable unknown unicast flooding
+          type: bool
+        pt2ptgem-flooding:
+          description:
+          - enable flooding on unicast GEM port
+          type: bool
+        mac-movement-ctrl:
+          description:
+          - enable mac movement in this vlan
+          type: bool
+        cvlan4095passthru:
+          description:
+          - enable C-VLAN 4095 tunneling behavior. Only applicable for S-VLAN CC forwarder
+          type: str
+          choices:
+          - passthru
+          - not-applicable
+        arp-snooping:
+          description:
+          - enable arp snooping
+          type: bool
+        arp-polling:
+          description:
+          - enable arp polling
+          type: bool
+        arp-polling-ip:
+          description:
+          - configure ARP polling IP address in form of 
+          - default = 0.0.0.0
+          type: str
+        mac-unauth:
+          description:
+          - enable mac unauthorized default : forward the frame to this vlan if authorization failed
+          type: bool
+    state:
+      description:
+      - The state the configuration should be left in.
+      type: str
+      choices:
+      - merged
+      - replaced
+      - overridden
+      - deleted
+options:
```

Comment:
Is this section part of the model?

Reply:
Oh. I seem to have modeled the behavior doubly. That was why I was surprised of the large amounts of options missing from this heavior. I have replaced the original model with the new model, which I believe is more complete.